recipe/meta.yaml

```diff
@@ -1,38 +1,56 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "3.0.0b2" %}
+{% set version_str = "v3.0.0-beta-2" %}
 
 package:
     name: protobuf
     version: {{ version }}
 
 source:
-    fn: protobuf-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.tar.gz
-    md5: 42aa7ab79ebcb94917e938ff482ef7a6
+    fn: protobuf_{{ version }}.tar.gz
+    url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
+    md5: e7f2602baffcbc27fb607de659cfbab6
 
 build:
-    number: 2
+    number: 3
+    skip: true  # [win]
 
 requirements:
     build:
+        - autoconf
+        - automake
+        - libtool
+        - pkg-config
         - python
-        - setuptools
+        - six
+        - ordereddict        # [py26]
+        - unittest2          # [py26]
+        - gcc                # [osx]
```

Comment:
Sorry, I have tried all compiler flags including the ones you have included they don't work. None. Here is the run down since you seem to be curious.

Here is the story if I force gcc (which will do the same thing as you mentioned though I can add that flag too) with or without `-std=c++0x`. I see the following error ( https://github.com/google/protobuf/issues/347 ). Having the stdlib flag doesn't matter here as gcc always uses stdlibc++, but I did try changing it anyways and still hit the same error. The proposed fix (which has nothing to do with gcc) is already in this release.

If I try to use `clang` and for `-stdlib=libc++`, I see this issue ( https://github.com/google/protobuf/issues/492 ). Again tried `-std=c++0x` and `-std=c++11`. As you can see, that issue affects people on both gcc/clang.

My impression remains the same. We are building a beta and it still has some kinks to work out. It doesn't seem to work on the system compilers. If this one works, let's use it. The gcc provided by conda was built on some ancient Mac OS, but I forget which.

As for your case, I am not surprised it works out of the box as they do CI on XCode 7.2. I'm guessing you are on the 7.x series somewhere so it should be close.

I agree that we should aim to get rid of it and move to a system compiler as soon as feasible hence my note above its use. However, we just can't get rid of it now. Honestly, I would love to use clang.